### PR TITLE
Fix workspace migration

### PR DESCRIPTION
**Pull request by yann300**
- add a "download file system" in the homepage
 - add a "migrate workspace" in the homepage and remove the migration at load

### CODE REVIEW

**Review by LianaHus — APPROVED** [review]

File: apps/remix-ide/src/app/ui/landing-page/landing-page.js [hunk 1]
```diff
@@ -357,6 +385,29 @@ export class LandingPage extends ViewPlugin {
       query.update({ appVersion: '0.7.7' })
       document.location.reload()
     }
+
+    const migrateWorkspace = async () => {
+      modalDialog(
+        'File system Migration',
+        yo`<span>'Do you want to download your files to local device first?'</span>`,
+        {
+          label: 'Download und Migrate',
```
Comment (Aniket-Engg) [inline apps/remix-ide/src/app/ui/landing-page/landing-page.js:394]: typo : `und` should be `and`

Comment (LianaHus) [reply apps/remix-ide/src/app/ui/landing-page/landing-page.js]: oops. it's german

**Review by ioedeveloper — CHANGES_REQUESTED** [review]

File: apps/remix-ide/src/app/ui/landing-page/landing-page.js [hunk 2]
```diff
@@ -357,6 +398,37 @@ export class LandingPage extends ViewPlugin {
...
+      try {
+        const workspaceName = await migrateToWorkspace(this.fileManager, this.filePanel)
+        tooltip('done. ' + workspaceName + ' created.')
+      } catch (e) {
+        return tooltip(e.message)
+      }
+    }
+
+    const migrateWorkspace = async () => {
+      modalDialog(
```
Comment (ioedeveloper) [inline apps/remix-ide/src/app/ui/landing-page/landing-page.js:413]: Clicking outside modal does not close the modal dialog. And when i click the `X` icon to close modal, i get a success message in the tooltip saying `done. workspace_migrated_1616584411034 created.`

**Review by ioedeveloper — CHANGES_REQUESTED** [review]

File: apps/remix-ide/src/app/ui/landing-page/landing-page.js (hunk 2 above)
Comment (LianaHus) [reply apps/remix-ide/src/app/ui/landing-page/landing-page.js:415]: In other pr

**Review by ioedeveloper — APPROVED** [review]

**Pull request merged**